### PR TITLE
Add tests for tag with emojis

### PR DESCRIPTION
**Pull request by davitol**
This PR adds tests for tagging with emojis and removes a duplicated test

### CODE REVIEW

File: tests/integration/features/tags.feature [hunk 1]
```diff
@@ -41,6 +50,14 @@ Feature: tags
     And The following tags should exist for "admin"
       |AnotherTagName|true|true|
 
+  Scenario: Renaming a normal tag to an emoji as regular user should work
+    Given user "user0" exists
+    Given "admin" creates a "normal" tag with name "MySuperAwesomeTagName"
+    When "user0" edits the tag with name "MySuperAwesomeTagName" and sets its name to "😀""
```
Comment (PVince81) [inline tests/integration/features/tags.feature]: extra double quote ?

**Comment by PVince81** [comment]
You were unlucky and one of the dockers didn't start.

Please rebase to rerun the test and use the opportunity to squash as well.

**Comment by PVince81** [comment]
```
14:41:18 --- Failed scenarios:
14:41:18 
14:41:18     /var/lib/jenkins/workspace/owncloud-core_core_PR-27406-RUIIGTWZWJ62JGNTDRIPG3WZQKHDORITZHNHNCK5YMCEK37OZ7WQ/tests/integration/features/tags.feature:53
```

**Comment by SergioBertolinSG** [comment]
This requires the same improvement as https://github.com/owncloud/core/pull/27332

**Comment by davitol** [comment]
👍

**Pull request merged**